app/controllers/application_controller.rb

```diff
@@ -2,4 +2,16 @@ class ApplicationController < ActionController::API
   rescue_from ActionController::ParameterMissing do
     render nothing: true, status: :bad_request
   end
+
+  private
+
+  def require_authorization!
+    head :unauthorized unless current_user
+  end
+
+  def current_user
+    return @current_user if defined? @current_user
+    user_id = request.headers['Authorization']
```

Comment:
This is just to get us off the ground. Once we have proper login support, we'll be providing proper auth tokens here. For now, we'll allow requests to claim to be any user by providing an existing user's ID.